### DISCUSSION TITLE
AVRO serialization DocArray

### DISCUSSION THREAD

**Discussion by nicklausbrown** [docarray/docarray#982] [DocArray v2]
Hello @JohannesMessner,

I was curious if you all would consider adding the AVRO serialization format for DocArray as well? It is quite common that we would like to serialize documents efficiently to and from Kafka topics. Thank you for the consideration!

**Comment by JohannesMessner** [comment]
Hey @nicklausbrown,
AVRO wasn't on our radar so far, but we have looked into it a bit, and it seems like an interesting format to support.
We just have a bit of a question mark around how well it plays with unstructured data.

Would you be interested in (at some point) contributing this feature, since you likely have more expertise than us? I would be happy to get you up to speed with the codebase etc.

@samsja @alaeddine-13 @JoanFM anything you guys wanna add?
